src/test/java/com/xceptance/xlt/engine/CachingHttpWebConnectionTest.java

```diff
@@ -164,36 +154,35 @@ public void testExpires_CacheControlMustRevalidate() throws Exception
     public void testExpires_CacheControlPublic_MaxAge() throws Exception
```

Comment:
:thinking:  Isn't this the same as `testExpires_CacheControlMaxAge` ? AFAIK there is no distinction between private and public.

Reply:
Removed.